ci/scripts/lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -eux
+
+pushd dp-upload-service
+  go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
```

Comment:
latest version is...?

```suggestion
  go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53.3
```